react-client/src/components/Pet/PetCardInfo.tsx

```diff
@@ -0,0 +1,49 @@
+import { PetInfo } from "~/mocks/MockRestApiServer";
+import { Button, Icon, Title } from "../design-system";
+
+export const PetCardInfo = ({ ...petInfo }: PetInfo) => {
+  const { breed, microchipNumber, name, sex } = petInfo;
+
+  return (
+    <div className="w-full p-[24px] lg:p-[40px]">
+      <div className="mb-[8px] flex max-h-[48px] w-full items-center justify-between">
```

Comment:
Prefer to use our spacing tokens instead of fixed pixels.

```suggestion
      <div className="mb-small flex max-h-[48px] w-full items-center justify-between">
```